OlinkAnalyze/R/olink_normalization_utils.R

```diff
@@ -1430,7 +1440,7 @@ olink_norm_input_clean_assays <- function(lst_df,
       df |>
         dplyr::filter(
           grepl(
-            pattern = "^OID\\d{5}_OID\\d{5}$",
+            pattern = "^OID\\d{5}_OID\\d{5}$|^OID\\d{5}$",
```

Comment:
This change makes sense in the case that a matched OlinkID is not found in the mapping file. See lines 1029-1032 and 1056-1059. Looks good.